### PR TITLE
Feature/admin tweaks

### PR DESCRIPTION
**Pull request by loneil**
# Description
- Launch Form button
- Couple minor bug fixes
- Preview Draft link on Current Draft text
- Disable create new button if existing draft

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-118

File: app/frontend/src/components/forms/manage/ManageVersions.vue [hunk 1]
```diff
@@ -49,18 +49,23 @@
...
-              color="primary"
-              text
-              small
-              @click="createVersion(item.formId, item.id)"
-              v-bind="attrs"
-              v-on="on"
-            >
-              <v-icon>add_circle</v-icon>
-            </v-btn>
+            <div v-bind="attrs" v-on="on">
```
Comment (TimCsaky) [inline app/frontend/src/components/forms/manage/ManageVersions.vue:52]: clicking this 'add' button isnt working for me. maybe these attributes need to be on the v-btn?

File: app/frontend/src/components/admin/AdministerUser.vue [hunk 2]
```diff
@@ -1,7 +1,18 @@
 <template>
   <div>
-    UI TBD
+    <h3>{{ user.fullName }}</h3>
+    <h4>User Details</h4>
     <pre>{{ user }}</pre>
+
+    <v-btn
+      color="primary"
```
Comment (TimCsaky) [inline app/frontend/src/components/admin/AdministerUser.vue:8]: nice idea!

File: app/frontend/src/components/forms/manage/ManageVersions.vue (hunk 1 above)
Comment (loneil) [reply app/frontend/src/components/forms/manage/ManageVersions.vue:52]: yikes, local change I'd forgot to push that fixes this. Thanks!

**Comment by TimCsaky** [comment]
- On the page: admin > forms, in the table, i'm guessing the launch only allows admin to see forms they have form_read permissions on (or public forms). seems ok.
- The whole draft/version thing works really well now .. once that publish toggle is on each version it will be perfect

**Review by TimCsaky — APPROVED** [review]

**Comment by loneil** [comment]
@TimCsaky yeah that's right, it's just the link to the form without doing RBAC checks for now. So if you can't access the form normally it will fail to load even as an admin.

**Pull request merged**